erp/management/commands/outscraper_clean_closed.py

```diff
@@ -0,0 +1,69 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.utils import timezone
+from outscraper import ApiClient
+
+from erp.models import Erp
+
+
+class Command(BaseCommand):
+    help = "Check for closed ERPs from outscraper API"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--start_pk",
+            type=int,
+            required=False,
+            default=0,
+            help="Resume the check since this given ERP PK in our DB.",
+        )
+        parser.add_argument(
+            "--write",
+            default=False,
+            action="store_true",
+            help="Actually edit the database",
+        )
+
+        parser.add_argument(
+            "--nb_days",
+            type=int,
+            required=False,
+            default=60,
+            help="Check the ERPs which have not been checked in the last nb_days.",
+        )
+
+    def _delete_erp(self, existing_erp):
+        print(f"Delete permanently closed ERP: {existing_erp}")
+        if self.write:
+            existing_erp.delete()
+        else:
+            print("Dry run mode, no DB action, use --write to apply this deletion")
+
+    def handle(self, *args, **options):
+        self.write = options["write"]
+        self.start_pk = options.get("start_pk")
+
+        client = ApiClient(api_key=settings.OUTSCRAPER_API_KEY)
+
+        two_month_ago = timezone.now() - timedelta(days=options["nb_days"])
```

Comment:
With the new parameter the variable has an improper naming I believe (proposals: date_limit, date_nb_days_ago